Doc/library/inspect.rst

```diff
@@ -526,12 +527,14 @@ Retrieving source code
 
 .. function:: getmodule(object)
 
-   Try to guess which module an object was defined in.
+   Try to guess which module an object was defined in. Return ``None``
+   in case of an error.
```

Comment:
Not necessarily an error; for example it returns None for primitive objects like ints that don't have a `__module__`.
```suggestion
   if the module cannot be determined.
```